### DISCUSSION TITLE
Is there an example of proper Reinforcement/Q Learning using Brain.js?

### DISCUSSION THREAD

**Discussion by flesler** [BrainJS/brain.js#745] [Q&A] [answered]
I'd like to make a browser simulation of an agent and have it get rewards based on what it does to train it.
I'm trying to find a good NN library for it that supports GPU, so Brain.js seems like a good candidate. But I couldn't find much about doing Q Learning with it. Would appreciate any input. Thanks in advance

**Comment by robertleeplummerjr** [comment, marked as answer]
Not at this time.  Though that is a goal in the future.
